packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -169,5 +169,25 @@ describe('build-schema', () => {
         description: 'test',
       });
     });
+
+    it('keeps AJV keywords', () => {
+      expect(
+        metaToJsonProperty({
```

Comment:
This statement is way too long to my taste, it also combines to parts of a test (act + assert) into a single line, which can make the test more difficult to understand. Let's split this into two statements please.

```ts
const schema = metaToJsonProperty({
  // ...
});

expect(schema).to.eql({
  // ...
});
```